VK/VK/Presentation/Profile/ViewControllers/ProfileViewController.swift

```diff
@@ -18,6 +18,9 @@ final class ProfileViewController: UIViewController {
         static let animateTransformScaleY = 0.6
         static let animateDisappearingProfileImageViewLayerOpacity: Float = 0.2
         static let animateEmergingProfileImageViewLayerOpacity: Float = 1
+        static let unknownFailureName = "Неизвестная ошибка"
+        static let decodingFailureName = "Ошибка декодирования"
+        static let urlFailureName = "Ошибка URL"
     }
```

Comment:
```suggestion
        static let unknownFailureName = "Неизвестная ошибка"
        static let decodingFailureName = "Ошибка декодирования"
        static let urlFailureName = "Ошибка URL"
    }
```
вот это константы четыре раза у тебя написаны) попробуй выносить в глобальный файл или даже лучше вот так например
public enum NetworkBaseState {
	case content
	case error
	case loading
	
	public var description: String {
		switch self {
		case .content:
			return "content"
		case .loading:
			return "loading"
		case .error:
			return "error"
		}
	}
}